### PR TITLE
use normalize_key instead of deprecated namespaced_key

### PR DESCRIPTION
**Pull request by dreyks**
This was already started in previous pulls.
`RedisStore#normalize_key` is already defined conditionally for backwards compatibility, so it's safe to simply replace all the occurrences of `namespaced_key` with `normalize_key` call

### CODE REVIEW

**Comment by dreyks** [comment]
all :green_heart:

**Comment by greysteil** [comment]
Would love to see this merged now Rails 5.0 is at release candidate stage.

File: test/active_support/cache/redis_store_test.rb [hunk 1]
```diff
@@ -317,7 +317,7 @@ def setup
 
   it "fetches data with expiration time" do
     with_store_management do |store|
-      store.fetch("rabbit", :force => true) # force cache miss
+      store.fetch("rabbit", :force => true) { @white_rabbit } # force cache miss
```
Comment (tubbo) [inline test/active_support/cache/redis_store_test.rb]: why did you have to change this test? seems strange.

**Comment by tubbo** [comment]
@dreyks I can merge this if you respond to that comment and rebase this branch. The implementation looks good, just questioning why you needed to change a seemingly unrelated test in order to get the suite passing.

**Comment by dreyks** [comment]
I changed this because it failed with the message that fetch requires a block

I'll rebase today and ping you

**Comment by dreyks** [comment]
oops, turns out that I accidentally deleted my fork... have to make another pull request.

**Pull request closed without merging**